src/BlockTypes/Cart.php

```diff
@@ -40,6 +40,14 @@ protected function initialize() {
 	public function register_patterns() {
 		$shop_permalink = wc_get_page_id( 'shop' ) ? get_permalink( wc_get_page_id( 'shop' ) ) : '';
 
+		register_block_pattern(
+			'woocommerce/cart-heading',
+			array(
+				'title'    => '',
+				'inserter' => false,
+				'content'  => '<!-- wp:heading {"align":"wide", "level":1} --><h1 class="wp-block-heading alignwide">' . esc_html__( 'Cart', 'woo-gutenberg-products-block' ) . '</h1><!-- /wp:heading -->',
```

Comment:
According to https://developer.wordpress.org/block-editor/reference-guides/components/heading/, the default heading level is 2. Using `<!-- wp:heading {"align":"wide"} -->` instead of `<!-- wp:heading {"align":"wide", "level":1} -->` would lead to the following error in the console:

<img width="785" alt="Screenshot 2023-07-20 at 12 50 27" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/b6122478-3059-465f-bd6f-470dc62b4bf0">

Reply:
We probably should use h1, because that is the previous heading for the page title, to keep things equivalent

Reply:
> We probably should use h1, because that is the previous heading for the page title, to keep things equivalent

That's what I'm already using:

```html
'<!-- wp:heading {"align":"wide", "level":1} --><h1 class="wp-block-heading alignwide">' . esc_html__( 'Cart', 'woo-gutenberg-products-block' ) . '</h1><!-- /wp:heading -->'
```

With my previous statement, I meant that I had to add `"level":1` to `<!-- wp:heading {"align":"wide", "level":1} -->`, so that WordPress expects to receive an `<h1>` element. Otherwise, WordPress expects to receive an `<h2>` element, while the template provides an `<h1>` element.